src/test/java/lesson3/task5/TriangleTest.java

```diff
@@ -0,0 +1,19 @@
+package lesson3.task5;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TriangleTest {
+
+    @Test
+    void shouldCheckRightTriangle() {
+        //given
+        Triangle rightTriangle = new Triangle(3, 4, 5);
```

Comment:
ok what if will create a new Triangle(5, 4, 3) - will it pass the test? if not why?

Reply:
Should not pass the test. As the rule follow the Pythagorean theorem where the square of sideC has to be equals the sum of the square of sideA with the square of sideB. Not sure if i should rename the sides to be more readable, but from the Wikipedia link, they do relate side A and B being the legs and side C the hypotenuse.